src/DataStructures/Tensor/Expressions/Evaluate.hpp

```diff
@@ -28,20 +28,20 @@ namespace TensorExpressions {
  * Given two rank 2 Tensors `R` and `S` with index order (a, b), add them
  * together and generate the resultant LHS Tensor `L` with index order (b, a):
  * \code{.cpp}
- * auto L = TensorExpressions::evaluate<ti_b_t, ti_a_t>(
+ * auto L = TensorExpressions::evaluate<ti_b, ti_a>(
  *     R(ti_a, ti_b) + S(ti_a, ti_b));
  * \endcode
  * \metareturns Tensor
  *
  * This represents evaluating: \f$L_{ba} = \R_{ab} + S_{ab}\f$
  *
- * @tparam LhsTensorIndices the TensorIndex of the Tensor on the LHS of the
- * tensor expression, e.g. `ti_a_t`, `ti_b_t`, `ti_c_t`
+ * @tparam LhsTensorIndices the TensorIndexs of the Tensor on the LHS of the
+ * tensor expression, e.g. `ti_a`, `ti_b`, `ti_c`
  * @tparam T the type of the RHS TensorExpression
  * @param rhs_te the RHS TensorExpression to be evaluated
  * @return the LHS Tensor with index order specified by LhsTensorIndices
  */
-template <typename... LhsTensorIndices, typename T,
+template <auto&... LhsTensorIndices, typename T,
```

Comment:
does just `auto` instead of `auto&` not work? I don't know enough about this to be able to give good advice, but `auto` seems like what we would want

Reply:
Ah yea... Right, it's not a feature until C++20. Could you add that as a detail in the documentation of `evaluate`? Just a statement that pre-C++20 we can't have classes as template parameters?

Reply:
Will do!

Reply:
Updated